fe/fe-core/src/main/java/com/starrocks/connector/hive/CacheUpdateProcessor.java

```diff
@@ -160,14 +160,28 @@ public void invalidateAll() {
     }
 
     public void invalidateTable(String dbName, String tableName) {
-        String tableLocation = ((HiveMetaStoreTable) metastore.getTable(dbName, tableName)).getTableLocation();
+        String tableLocation;
+        try {
+            tableLocation = ((HiveMetaStoreTable) metastore.getTable(dbName, tableName)).getTableLocation();
+        } catch (Exception e) {
+            LOG.warn("Failed to get table {}.{}. ignore it", dbName, tableName);
+            return;
```

Comment:
Just return, the table cache or partition cache may still exists?

Reply:
yes, this happens during the restart. the cache is empty.  if delete the hive table first and then delete the hive external table. we will get table from cache. if the cache miss, others cache entry of this table are also empty.